src/types/notice.ts

```diff
@@ -288,12 +297,71 @@ export default /* GraphQL */ `
   }
 
   enum CircleNoticeType {
-    CircleNewFollower # deprecated
     CircleNewSubscriber
+    CircleNewFollower
     CircleNewUnsubscriber
     CircleInvitation
+    CircleNewDiscussion
```

Comment:
though on design these comment-related circle notices don't show comment in UI, I prefer to use `CircleCommentNotice` instead, and store comment FK in the notice table (in case of product changes).

Reply:
if it carries the comment detail, then seems unable to bundle